students/jarosz_natalia/lesson_03/calculator.py

```diff
@@ -0,0 +1,89 @@
+print("Welcome to badly organized calculator:")
```

Comment:
I this line could be moved to the line 48 before ```while``` statement, Than in line 49 you can call ```help()``` and than lines from 2 to 8 could be deleted:)